### PR TITLE
Issue/34

### PR DESCRIPTION
**Pull request by mik01aj**
These are some structural changes that I needed to write my test in #34. The functionality of Styleguidist should NOT change in any way.

### CODE REVIEW

File: loaders/styleguide.loader.js [hunk 1]
```diff
@@ -89,6 +89,13 @@ module.exports = function() {};
 module.exports.pitch = function() {
 	this.cacheable && this.cacheable();
 
+	if (this.query) {
```
Comment (sapegin) [inline loaders/styleguide.loader.js]: Could you add a comment with an example of loader usage with this option?

**Comment by mik01aj** [comment]
@sapegin I could merge it myself, but I prefer to wait for your feedback. If there's none, I'll merge this tomorrow.

File: loaders/styleguide.loader.js (hunk 1 above)
Comment (mik01aj) [inline loaders/styleguide.loader.js]: ok, will add

**Comment by mik01aj** [comment]
@sapegin thank you! could you make a new release for me? :)

**Comment by mik01aj** [comment]
thank you! 👍

**Pull request merged**